### PR TITLE
Kibana enhanced table plugin installation steps

### PR DESCRIPTION
**Pull request by Avarjana**
Kibana enhanced table plugin is used to overcome the issue https://github.com/wso2/product-is/issues/14127. Therefore, the documentation is altered to include the installation steps for the plugin.

### CODE REVIEW

File: en/docs/deploy/elk-analytics-installation-guide.md [hunk 1]
```diff
@@ -101,18 +101,22 @@ Follow the steps below to enable ELK-based analytics in WSO2 Identity Server.
  
 ## Configure ELK Analytics Dashboards
  
-1. Log in to Kibana.
-2. Navigate to **Stack Management** > **Index Management** and select the **Index Templates** tab.
-3. Under the **Index patterns** column, if you have any index patterns created under the following names, delete them before moving to the next step.
+1. Navigate to Kibana installation folder and run the following command to install [Kibana Enhanced Table](https://github.com/fbaligand/kibana-enhanced-table) plugin. Replace A,B,C with the plugin version and X,Y,Z with the Kibana version you have installed.
```
Comment (himeshsiriwardana) [inline en/docs/deploy/elk-analytics-installation-guide.md:104]:
```suggestion
1. Navigate to Kibana installation folder and run the following command to install the [Kibana Enhanced Table](https://github.com/fbaligand/kibana-enhanced-table) plugin. Replace A,B,C with the plugin version and X,Y,Z with the Kibana version you have installed.
```

Comment (himeshsiriwardana) [inline en/docs/deploy/elk-analytics-installation-guide.md:104]: Let's have 'Replace A,B,C with the plugin version and X,Y,Z with the Kibana version you have installed.' in an info box.

Comment (Avarjana) [reply en/docs/deploy/elk-analytics-installation-guide.md]: Addressed in https://github.com/wso2/docs-is/pull/3137/commits/2ea76a06be01025bb2a5822255f5a54ee5fe9d3b

Comment (Avarjana) [reply en/docs/deploy/elk-analytics-installation-guide.md]: Addressed in https://github.com/wso2/docs-is/pull/3137/commits/2ea76a06be01025bb2a5822255f5a54ee5fe9d3b

**Review by himeshsiriwardana — APPROVED** [review]

**Pull request merged**